packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -348,12 +357,15 @@ const TESTS: [string, any, Record<string, any>?][] = [
       ['X-Verify-Code', { test: 'equal', expected: MESSAGE.code }],
     ])],
     ['html', [
+      { test: 'include', expected: 'Reminder: Finish creating your account' },
```

Comment:
I took several `subject`s in FxA and SubPlat emails and tested the HTML for them like here. This checks that they're in the layout files as expected, which means the `subject` is in `<mj-title>` which will show the subject as the tab title in some browsers.

While I was looking through these, I added a few more tests, like in `lowRecoveryCodes` to make sure the `includes.ts` file is giving us what we expect it to. Additional Renderer / Localizer specific tests will be added when #11960 is complete (either here or in another PR).